xwalk.gyp

```diff
@@ -715,6 +734,11 @@
             '../base/allocator/allocator.gyp:allocator',
           ],
         }],
+        ['OS=="win" and enable_widevine==1', {
+          'dependencies': [
+            '../third_party/widevine/cdm/widevine_cdm.gyp:widevinecdmadapter',
```

Comment:
@darktears @pozdnyakov Sorry for the mislead to download the <code>widevinecdmadapter.dll</code>. The above code in gyp <code>'../third_party/widevine/cdm/widevine_cdm.gyp:widevinecdmadapter',</code> will produce a <code>widevinecdmadapter.dll</code> file when you use <code>ninja</code> build. I have modified the description info. Please check again. About @pozdnyakov 's suggestion, I will send a 'intend to implement' soon. Thanks.